src/playlist-selectors.js

```diff
@@ -197,40 +197,20 @@ export const simpleSelector = function(
   // filter out playlists without resolution information
   const haveResolution = bandwidthPlaylistReps.filter((rep) => rep.width && rep.height);
 
-  // sort variants by resolution
-  stableSort(haveResolution, (left, right) => left.width - right.width);
-
-  // if we have the exact resolution as the player use it
-  const resolutionBestRepList = haveResolution.filter((rep) => rep.width === playerWidth && rep.height === playerHeight);
```

Comment:
We don't need this now that we use `pixelDiff` since the `pixelDiff` will be `0` for such a playlist.